src/integration-test/java/com/commercetools/sync/integration/externalsource/producttypes/ProductTypeSyncIT.java

```diff
@@ -457,7 +456,7 @@ public void sync_WithErrorUpdatingTheProductType_ShouldExecuteCallbackOnErrorAnd
         assertThat(errorMessages)
             .hasSize(1)
             .hasOnlyOneElementSatisfying(message ->
-                assertThat(message).contains("Failed to update product type of key 'key_1'.")
+                assertThat(message).contains("Failed to update product type of key: 'key_1'.")
```

Comment:
same meaning but `with key` or `of key` which is better for consistency?

Reply:
as you said, doesn't matter, but better be consistent. So now it should be consistently with key.